samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BasicSendReceiveUsingQueueClient
+{
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Primitives;
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        const string ServiceBusConnectionString = "{ServiceBus connection string}";
```

Comment:
We could potentially add a config file to the solution and then add links for that file in each of the projects.. that would work.. but then, I still dont want to hardcode queue names and force user to create one i have coded up, which means the User has to anyway update the queue name here in the particular file(since some tests use session based queues and some are non-session based). We could add queue names too to the config file, but then that means we have to explain to the user somewhere else that X tests uses Y as queue Name and A Test uses B as queueName. So please update that accordingly.. This looked better and easy to explain to a new person starting to try out the samples?